docusaurus/docs/reference/ha/bootstrapping.md

```diff
@@ -0,0 +1,164 @@
+---
+sidebar_label: Bootstrapping
+sidebar_position: 10
+---
+
+# Bootstrapping A Cluster 
+
+To bring up a controller cluster, one starts with a single node. 
+
+## Controller Configuration
+
+### Certificates
+
+Each controller requires appropriate certificates. The certificates for clustered controllers 
+have more requirements than those for a standalone server. See the [Certificates Reference](./certificates.md)
+for more information. 
+
+### Config File
+
+The controller requires a `raft` section.
+
+```yaml
+raft:
+  dataDir: /path/to/data/dir
+```
+The `dataDir` will be used to store the following:
+
+* `ctrl-ha.db` - the ziti model bbolt database
+* `raft.db` - the raft bbolt database
+* `snapshots/` - a directory to store raft snapshots
+
+Controller use the control channel listener to communicate with each other. Unlike
+routers, they need to know how to reach each other, so an advertise address must
+be configured.
+
+```yaml
+ctrl:
+  listener: tls:0.0.0.0:6262
+  options:
+    advertiseAddress: tls:192.168.1.100:6262
+```
+ 
+Finally, for sessions to work across controllers, JWTs are used. To enable these
+an OIDC endpoint should be configured.
+
+```yaml
+web:
+  - name: all-apis-localhost
+    bindPoints:
+      - interface: 127.0.0.1:1280
+        address: 127.0.0.1:1280
+    options:
+      minTLSVersion: TLS1.2
+      maxTLSVersion: TLS1.3
+    apis:
+      - binding: health-checks
+      - binding: fabric
+      - binding: edge-management
+      - binding: edge-client
+      - binding: edge-oidc
+```
+
+## Initializing the Controller
+
+Once properly configured, the controller can be started.
```

Comment:
> Once properly configured

Makes me think -- this will be easy to misconfigure. If there are misconfigurations, a troubleshooting guide would be good to see. I suspect there's not one yet so maybe just say "start the controller"?